content/deploy-troubleshooting.md

```diff
@@ -7,18 +7,24 @@ description: Learn how to troubleshoot your deploy and get answers to frequently
+* Your container's memory and CPU usage. If your app is running out of memory, you may need to switch to a larger container, use more containers, or refactor your app to use less memory.
+  * In the short term, the only guaranteed solution is to scale up and use enough Galaxy resources to handle all your app's memory needs.
+* Your app's logs. While 'All' shows all output, consider breaking it down by tab. If your app is running and struggling before failing, check the 'App' tab. If your app fails when Galaxy tries to build it into a container image, check the 'Service' tab.
+  * Note that both can happen simultaneously: the earlier version of your app may be throwing errors, while the recent version created to fix the problem may have a code issue preventing deployment. In that case, the 'Service' tab will be more helpful.
+* Check <a href="http://github.com/meteor/meteor/issues/">GitHub</a> to see if any related Meteor issue lists workarounds or solutions.
+* Check the <a href="https://forums.meteor.com/">forums</a> for related issues and solutions. 
+* Consider running more than 1 container, or 3 containers to qualify for high-availability status.
+  * If you run one container, that makes the machine your container is running on a single point of failure. In the event of a hardware failure, your app will be down until Galaxy starts it up again on a new machine.
+* Write in to <a href="mailto:support@meteor.com">support</a>. To minimize the back-and-forth, please send in the name of the affected app, the conditions that trigger the issue (confirmed or suspected), steps to reproduce, and relevant logs. Try to resolve errors listed in the logs before writing in.
+  * If your app's container is running with the error, please try to leave it in the running state when contacting support, since that will help us to troubleshoot. Do not remove containers throwing errors, as that will prevent us from inspecting the container and the logs.
+  * While stopping the container is not as helpful for troubleshooting as leaving the container running, it will leave the logs intact.
 
 Note that code-level review lies outside the scope of Galaxy's support. If this is important to you, consider [Meteor Development Support](/support.html).
 
-<h2 id="503-errors">503 errors or Deployment Failures</h2>
+<h2 id="503-errors">503 errors, 502 errors or Deployment Failures</h2>
 
-Your app may throw a 503 error and show `Service Unavailable: No healthy endpoints to handle the request` when you try to visit your URL. This means no healthy containers are currently available to serve your app.
+Your app may throw a 503 error and show `Service Unavailable: No healthy endpoints to handle the request` when you try to visit your URL. You may also see a 502 error with the message `Registered endpoints failed to handle the request.` This means no healthy containers are currently available to serve your app.
```

Comment:
502 is actually different.  502 means that the request failed despite the fact that we thought there was a healthy container at the beginning of the request, perhaps because the backend never responded to our proxy.